### ISSUE TITLE
Page should start with an H1 tag (and headings should be not out of order)

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/purl#755]
On the [PURL landing page](https://purl.stanford.edu/) (only page affected), **About Persistent URLs (PURLs)** should be an H1 instead of an H2. And since headings shouldn't be out of order or skip a level, that means that ### Featured Items should be an H2 instead of an H3.

Reference: https://www.w3.org/WAI/tutorials/page-structure/headings/

<img width="1062" alt="Screenshot 2023-10-31 at 1 22 53 PM" src="https://github.com/sul-dlss/purl/assets/6816624/82bffdcf-3741-4b90-8faa-3724237d713d">
